src/main/java/com/spotify/docker/client/DockerClient.java

```diff
@@ -2332,10 +2355,33 @@ public static EventsParam label(final String label, final String value) {
      *
      * @param label The label to filter on
      * @return EventsParam
+     * @since API 1.21
      */
     public static EventsParam label(final String label) {
       return label(label, null);
     }
+
+    /**
+     * Valid event types for EventsParam.type
+     */
+    public enum EventType {
```

Comment:
Let me think this through. I'm not sure exactly what you’re asking to test.

Do you mean we should test whether the `filter=type=<>` param works correctly, and that if we ask for, say, only image events, then that's all we get back? That's doable. A test could do a bunch of stuff—pull an image, create a container, create a volume, etc.—then ask docker for the different types of events one-by-one, and verify that we only get back what we expect.

Or do you mean to test the validity of the `enum` itself, that it has all the values that it needs to have? I don't know how that would be tested. I'm kind of trusting that the API fully documents all the valid values.